src/Framework/Sources/JavaScript/Content/chrome/storage.js

```diff
@@ -0,0 +1,50 @@
+
+'use strict';
+
+const background = require('../background-bridge');
+
+var storage = {
+    /**
+     * @param keys (optional)
+     * @param cb function
+     */
+    get (keys, cb) {
+        const callback = cb || keys;
+        background.dispatchRequest(
+            {
+                eventName: 'storage.get',
+                message: {
+                    keys
+                }
+            },
+            (resp) => callback(resp)
+        );
+    },
+    set(items) {
+        background.dispatchRequest(
+            {
+                eventName: 'storage.set',
+                message: {
+                    items
+                }
+            }
+            // (resp) => callback(resp)
+        );
+    }
+};
+
+module.exports = {
+    local: storage,
+    sync: storage,
+    managed: {
+        get() {
+            return new Promise(function (resolve) {
```

Comment:
please get rid of the promise. this has already costed me some debugging time figuring out why an extension is stuck.

Reply:
Sorry about that. It was added when i tried to inject languagetooler right into TopeeDemo and i forgot to remove it.
Removed